lib/client/wallet.js

```diff
@@ -325,6 +325,16 @@ class WalletClient extends Client {
     return this.post(`/wallet/${id}/zap`, { account, age });
   }
 
+  /**
+   * @param {Number} id - Wallet id.
+   * @param {Number} hash - Tx hash.
```

Comment:
Note that its expected to be in big endian?

Reply:
I don't think we generally do that.

Reply:
However it should probably say Buffer|Hash for the jsdoc.

Reply:
I think that we should start to document endianness. Its a gotcha for users and developers.

Reply:
FWIW Looking for over this file, and there are various jsdocs missing...

Reply:
...for example `id` is never mentioned in the params.

Reply:
I'll just add that now.